tools/dep_updaters/update-cares.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/sh
+set -e
+# Shell script to update c-ares in the source tree to a specific version
+
+BASE_DIR=$(cd "$(dirname "$0")/../.." && pwd)
+DEPS_DIR="$BASE_DIR/deps"
+
+[ -z "$NODE" ] && NODE="$BASE_DIR/out/Release/node"
+[ -x "$NODE" ] || NODE=$(command -v node)
+
+NEW_VERSION="$("$NODE" --input-type=module <<'EOF'
+const res = await fetch('https://api.github.com/repos/c-ares/c-ares/releases/latest');
+if (!res.ok) throw new Error(`FetchError: ${res.status} ${res.statusText}`, { cause: res });
+const { tag_name } = await res.json();
+console.log(tag_name.replace('cares-', '').replaceAll('_', '.'));
+EOF
+)"
+
+CURRENT_VERSION=$(grep "#define ARES_VERSION_STR" ./deps/cares/include/ares_version.h |  sed -n "s/^.*VERSION_STR \"\(.*\)\"/\1/p")
+
```

Comment:
Can you add a echo "Comparing $NEW_VERSION with $CURRENT_VERSION" for debugging purposes?